addon/index.js

```diff
@@ -20,16 +19,13 @@ const { set, reset } = MockDate || {
  * https://github.com/BackburnerJS/backburner.js/pull/264
  */
 const freezeDateAt = (...args) => {
-  trySet(Ember, 'run.backburner._platform.now', () => originalDate());
+  trySet(run, 'backburner._platform.now', originalDate.now());
```

Comment:
It seems like backburner tries to call this as a function, so now should be passed here without calling it.

```suggestion
  trySet(run, 'backburner._platform.now', originalDate.now);
```